src/applications/proxy-rewrite/partials/header.js

```diff
@@ -0,0 +1,80 @@
+export default `
```

Comment:
@ncksllvn the partial you provided in https://github.com/department-of-veterans-affairs/vets-website/pull/8295/ wasn't from the va gov content directory. Since those are using react I figured we'd give that integration a shot first and address performance issues as they pop up.

Reply:
Also not quite sure how we can best get those into the static pages... this seemed like a good first pass. Just need to validate the approach (or start thinking _really_ hard about other options)...